tests/test_rse.py

```diff
@@ -532,8 +532,8 @@ def test_add_protocol(self, vo, rucio_client):
                       'prefix': '/the/one/with/all/the/files',
                       'impl': 'rucio.rse.protocols.SomeProtocol.SomeImplementation',
                       'domains': {
-                          'lan': {'read': 4,
-                                  'write': 1,
+                          'lan': {'read': 2,
```

Comment:
Same comment here, read went from 4 to 2 instead of 3 (I assume it's fine as we only have 3 protocols)